File: CSharp/1048 - Aumento de Salário.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    double A = Convert.ToDouble(Console.ReadLine());
    double B;
    int C;
    if (A <= 400.00){
      B = A * 1.15;
      C = 15;
    }
    else if (A <= 800.00){
      B = A * 1.12;
      C = 12;
    }
    else if (A <= 1200.00){
      B = A * 1.10;
      C = 10;
    }
    else if (A <= 2000.00){
      B = A * 1.07;
      C = 7;
    }
    else{
      B = A * 1.04;
      C = 4;
    }
    Console.WriteLine("Novo salario: " + B.ToString("0.00"));
    Console.WriteLine("Reajuste ganho: " + (B - A).ToString("0.00"));
    Console.WriteLine("Em percentual: " + C + " %");
  }
}

File: Python 3.8/1048 - Aumento de Salário.py
x = float(input())
if 0 <= x <= 400:
  reajuste = 0.15
elif 400 < x <= 800:
  reajuste = 0.12
elif 800 < x <= 1200:
  reajuste = 0.10
elif 1200 < x <= 2000:
  reajuste = 0.07
elif x > 2000:
  reajuste = 0.04

nreajuste = x * reajuste
salario = nreajuste + x
print('Novo salario: {:.2f}'.format(salario))
print('Reajuste ganho: {:.2f}'.format(nreajuste))
print('Em percentual: {:.0f} %'.format(reajuste * 100))
